sql/telemetry/clients_scalar_aggregates_v1/query.sql

```diff
@@ -0,0 +1,161 @@
+WITH latest_versions AS (
+  SELECT channel, MAX(CAST(app_version AS INT64)) AS latest_version
+  FROM
+    (SELECT
+      normalized_channel AS channel,
+      SPLIT(application.version, '.')[OFFSET(0)] AS app_version,
+      COUNT(*)
+    FROM `moz-fx-data-shared-prod.telemetry_stable.main_v4`
+    WHERE DATE(submission_timestamp) > DATE_SUB(CURRENT_DATE(), INTERVAL 90 DAY)
+    AND normalized_channel IN ("nightly", "beta", "release")
+    GROUP BY 1, 2
+    HAVING COUNT(*) > 1000
+    ORDER BY 1, 2 DESC)
+  GROUP BY 1),
+
+filtered_date_channel AS (
+  SELECT *
+  FROM clients_daily_scalar_aggregates_v1
+  WHERE submission_date > DATE_SUB(CURRENT_DATE(), INTERVAL 90 DAY)
+  AND channel IN ('release', 'beta', 'nightly')
+),
+
+filtered_aggregates AS (
+  SELECT
+    submission_date,
+    client_id,
+    os,
+    app_version,
+    app_build_id,
+    channel,
+    metric,
+    metric_type,
+    key,
+    agg_type,
+    value
+  FROM filtered_date_channel
+  CROSS JOIN
+    UNNEST(scalar_aggregates)
+  WHERE value IS NOT NULL
+),
+
+version_filtered AS (
+  SELECT
+    client_id,
+    os,
+    app_version,
+    app_build_id,
+    scalar_aggs.channel AS channel,
+    metric,
+    metric_type,
+    key,
+    agg_type,
+    value
+  FROM filtered_aggregates AS scalar_aggs
+  LEFT JOIN latest_versions
+  ON latest_versions.channel = scalar_aggs.channel
+  WHERE CAST(app_version AS INT64) >= (latest_version - 2)
+),
+
+scalar_aggregates AS (
+  SELECT
+    client_id,
+    os,
+    app_version,
+    app_build_id,
+    channel,
+    metric,
+    metric_type,
+    key,
+    agg_type,
+    CASE agg_type
```

Comment:
Are we missing `count`? That's how we'll get `avg`: `sum / count`

Reply:
good catch! I'll update this. Thank you